src/ducks/devices.js

```diff
@@ -50,13 +50,13 @@ export const {
   action: deviceSyncUpdate,
   reducer: deviceSyncUpdateReducer
 } = create('DEVICE_SYNC_UPDATE', {
-  start: (state, meta: DeviceSyncUpdateMeta) => {
+  success: (state, meta, payload) => {
```

Comment:
this was simply not passed correctly from the other side, type checking wasn't really catching this for some reason